aws/idp/ecs.tf

```diff
@@ -64,7 +64,7 @@ module "idp_ecs" {
 
   # Task definition
   container_image       = "${var.zitadel_image_ecr_url}:${var.zitadel_image_tag}"
-  container_command     = ["start-from-init", "--masterkeyFromEnv", "--tlsMode", "enabled", "--config", "/app/config.yaml", "--steps", "/app/steps.yaml"] # TODO: update to `start` command only for prod
+  container_command     = ["start-from-init", "--masterkeyFromEnv", "--tlsMode", "disabled", "--config", "/app/config.yaml", "--steps", "/app/steps.yaml"] # TODO: update to `start` command only for prod
```

Comment:
I'm wondering if we should go with "external" instead?
https://zitadel.com/docs/self-hosting/manage/tls_modes#external

Reply:
I have seen that and I was wondering if it would require any other specific configuration or if implied something else that would have an impact somewhere else. Does not look like it but then why this option exist? It would be pretty much like `disabled`.

Reply:
Except that it tells the clients downstream to use https even though it is not aware of the TLS connection itself.

Reply:
We can test it though ;)